srcpkgs/arp-scan/template

```diff
@@ -9,8 +9,9 @@ short_desc="Tool that uses ARP to discover IP hosts on the local network"
 maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/royhills/arp-scan"
-distfiles="https://github.com/royhills/${pkgname}/archive/${version}/${pkgname}-${version}.tar.gz"
-checksum=e03c36e4933c655bd0e4a841272554a347cd0136faf42c4a6564059e0761c039
+changelog="https://raw.githubusercontent.com/royhills/arp-scan/master/NEWS.md"
+distfiles="https://github.com/royhills/arp-scan/archive/${version}/arp-scan-${version}.tar.gz"
+checksum=204b13487158b8e46bf6dd207757a52621148fdd1d2467ebd104de17493bab25
```

Comment:
I am not sure why xlint in CI didn't catch this, but this should be $distfiles, then $changelog, then $checksum.

Reply:
Are you sure? If i do `xnew pkgname`, then it is $changelog, $distfiles, $checksum

![image](https://user-images.githubusercontent.com/58117792/225665706-c9598640-fd79-43e5-b9da-0cb20081b1ca.png)